src/main/java/com/farao_community/farao/gridcapa/data_bridge/MinioSink.java

```diff
@@ -52,7 +52,7 @@ public MessageChannel minioSinkChannel() {
     @ServiceActivator(inputChannel = "filesChannel")
     public MessageHandler s3MessageHandler(FileMetadataProvider fileMetadataProvider) {
         S3MessageHandler s3MessageHandler = new S3MessageHandler(amazonS3(), bucket);
-        Expression keyExpression = new SpelExpressionParser().parseExpression("'" + baseDirectory + "/' + headers.file_name");
+        Expression keyExpression = new SpelExpressionParser().parseExpression("'" + baseDirectory + "' +  headers.file_sink + '/' + headers.file_name");
```

Comment:
Do you include the "/" inside the `file_sink` definition? Isn't it disturbing to define as a sort of "absolute path" whereas it is a relative path in reality?